style/CONTRIBUTE.md

```diff
@@ -124,3 +124,27 @@ restart the web server.
 
 Please prettify all files prior to submission. Run `make code_format` to format all
 code files with js-prettifier.
+
+## Highway Shield Contributor's Guide
+
+Highway shields are a key feature of the OpenStreetMap Americana style. This guide describes some of the style principles that contributors of highway shield artwork should consider when submitting new shields. The required elements are as follows:
+
+1. Shields are in .svg format, saved in "Optimized SVG" format in Inkscape
+2. Shields are 20px on the smallest dimension (this will be rasterized to 20px or 40x depending on display resolution)
+3. Shields are license-compatible (public domain or CC0)
+
+In addition, the following style guidelines are recommended:
+
+1. Use a 1.25px stroke size for line elements.
+2. Complex or intricate graphic elements should be simplified
+3. Background shading should be removed
+4. Route numbers should be large enough to be legible
+
+In general, this style is not trying to exactly replicate highway shields as seen on signage. Instead, we are trying to extract the key stylistic elements so that the graphics are recognizable as simplifications of their real-world counterparts. Here are some examples of Americana's simplified shields for small-size readability:
+
+| Network                | Real-world Shield                                                                                          | Americana Representation                                                                                                                    |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| NJ Turnpike            | <img src="https://upload.wikimedia.org/wikipedia/commons/a/af/New_Jersey_Turnpike_Shield.svg" width="40"/> | <img src="https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/style/icons/shield40_us_nj_njtp_noref.svg" width="40"/> |
+| Washington State Route | <img src="https://upload.wikimedia.org/wikipedia/commons/5/52/WA-blank.svg" width="40"/>                   | <img src="https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/style/icons/shield40_us_wa.svg" width="40"/>            |
+
+More complicated shields may be more challenging to simplify. Consider taking some inspiration from the [rebusurance](https://github.com/1ec5/rebusurance) project, which effectivly simplifies a number of complex state shield designs by stretching, compressing, omitting, or simplifying graphic elements.
```

Comment:
The very first step I did for the Rebusurance project was to adopt a unified color palette for every shield. Rebusurance uses the same standard [MUTCD colors](https://commons.wikimedia.org/wiki/Commons:MUTCD_colors) as Wikimedia Commons. These colors are effectively Americana’s color palette at the moment, because many of the shields were derived from Commons images, but we can choose a different color scheme that aligns with the rest of the map. Internal consistency is important because even slightly different shades of blue or green are easily perceptible on some screens. Shields from other countries should probably share the same color palette, with same exceptions for unusual colors like sky blue.

We could check in an [Inkscape color palette](https://commons.wikimedia.org/wiki/Commons:WikiProject_U.S._Roads/Shields#Installing_the_MUTCD_color_palette) for contributors to install. Maybe there’s even a tool out there for listing SVGs for compliance with this color palette that we could run on CI.

Reply:
This makes sense to me.  Other digital maps don't try to perfectly match shield colors so I would support a standard palette.  Also, the Inkscape color palette is a great idea.  However, I feel wildly incompetent in graphic design and would welcome help in the best way to document and provide such a palette.